src/coreclrDebug/activate.ts

```diff
@@ -222,12 +224,18 @@ function showInstallErrorMessage(eventStream: EventStream) {
 function showDotnetToolsWarning(message: string): void {
     const config = vscode.workspace.getConfiguration('csharp');
     if (!config.get('suppressDotnetInstallWarning', false)) {
-        const getDotNetMessage: ActionOption = {
-            title: vscode.l10n.t('Get the SDK'),
-            action: async () => {
-                await vscode.env.openExternal(vscode.Uri.parse('https://dot.net/core-sdk-vscode'));
-            },
-        };
+        const getDotNetMessage: ActionOption | Command =
+            getCSharpDevKit() !== undefined
+                ? {
+                      title: vscode.l10n.t('Get the SDK'),
+                      command: 'csdevkit.installDotnetSdk',
+                  }
```

Comment:
This extension does not use C# Dev Kit extension so this part should be removed:
```suggestion
            getCSharpDevKit() !== undefined
                ? {
                      title: vscode.l10n.t('Get the SDK'),
                      command: 'csdevkit.installDotnetSdk',
                  }
```